diff -Naur puppetdb-orig/install.sh puppetdb-patched/install.sh
--- puppetdb-orig/install.sh	2018-11-01 18:54:00.000000000 +0100
+++ puppetdb-patched/install.sh	2018-11-02 07:42:24.962361877 +0100
@@ -172,8 +172,6 @@
     install -d -m 0755 "${dest_apps_dir}/cli/apps"
     install -d -m 0755 "${DESTDIR}${bindir}"
     install -m 0755 "ext/bin/${real_name}" "${DESTDIR}${bindir}/${real_name}"
-    install -d -m 0755 "${DESTDIR}${symbindir}"
-    ln -s "../apps/${real_name}/bin/${real_name}" "${DESTDIR}${symbindir}/${real_name}"
     install -d -m 0755 "${DESTDIR}${uxbindir}"
     ln -s "../server/apps/${real_name}/bin/${real_name}" "${DESTDIR}${uxbindir}/${real_name}"
     install -m 0755 ext/cli/stop "${dest_apps_dir}/cli/apps/stop"
